src/core/tests/frontend/paddle/op_fuzzy.cpp

```diff
@@ -159,6 +159,14 @@ static const std::vector<std::string> models{
     std::string("gather_one_dimension2"),
     // gather_axis_input
     // (CVS-82724: not support Axis as input),
+    std::string("gather_nd_float32"),
+    std::string("gather_nd_int64"),
+    std::string("gather_nd_int32"),
+    // Not support Axis as input),
+    // std::string("gather_nd_empty"),
```

Comment:
```suggestion
    // not support index zero dimension,
    // std::string("gather_nd_empty"),
```
the description here is not so accurate, better change or remove it.

Reply:
done